tests/capture-tests.js

```diff
@@ -243,20 +243,17 @@ require(["mobifyjs/utils", "capture"], function(Utils, Capture) {
         $("#qunit-fixture").append($iframe);
     });
 
-    asyncTest("ios8_0ScrollFix", function() {
+    // A metatag should not be inserted if it was not present in the original doc
```

Comment:
Do you think it's useful to keep around a test case that maintains the original intent of this test? i.e. that this function properly injects a meta tag? (albeit, after your change, only when a meta tag previously existed in the original document)